tests/integration/test_api.py

```diff
@@ -420,31 +420,49 @@ def test_prediction_collection_no_filter(client):
 
 
 def test_get_predictions():
-    PredictionFactory(barcode="123", keep_types="category", value_tag="en:seeds")
-    PredictionFactory(type="category")
-    PredictionFactory(value_tag="en:seeds")
-    PredictionFactory(type="brand")
+    prediction1 = PredictionFactory(
+        barcode="123", keep_types="category", value_tag="en:seeds"
+    )
+    prediction2 = PredictionFactory(
+        barcode="123", keep_types="category", value_tag="en:beers"
+    )
+    prediction3 = PredictionFactory(
+        barcode="123", keep_types="label", value_tag="en:eu-organic"
+    )
+    prediction4 = PredictionFactory(
+        barcode="456", keep_types="label", value_tag="en:eu-organic"
+    )
 
     actual_prediction1 = get_predictions(barcode="123")
     actual_items1 = [item.to_dict() for item in actual_prediction1]
+    assert actual_items1[0]["id"] == prediction1.id
     assert actual_items1[0]["barcode"] == "123"
     assert actual_items1[0]["type"] == "category"
     assert actual_items1[0]["value_tag"] == "en:seeds"
-    assert len(actual_items1) == 1
+    assert actual_items1[1]["value_tag"] == "en:beers"
+    assert actual_items1[2]["value_tag"] == "en:eu-organic"
+    assert len(actual_items1) == 3
 
     actual_prediction2 = get_predictions(keep_types=["brand"])
     actual_items2 = [item.to_dict() for item in actual_prediction2]
-    assert actual_items2[0]["barcode"] == "123"
-    assert actual_items2[0]["type"] == "brand"
-    assert actual_items2[0]["value_tag"] == "en:seeds"
+    assert prediction2.id == prediction1.id + 1
```

Comment:
@alexgarel 

I am not sure about this line though

Reply:
you should not have to test that. First it's not related to your test (you are not supposed to be testing PredictionFactory here), and second it's not garanteed to be true.